CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Changelog
 
-## 1.X.X
+## [1.3.0]
 
-`2021-05-XX`
+`2021-05-10`
 
 ### What's new
 
```

Comment:
Maybe worth mentioning that random build/test failure should be fixed because of the icons package update?